app/javascript/controllers/application.js

```diff
@@ -3,7 +3,7 @@ import { Application } from "@hotwired/stimulus"
 const application = Application.start()
 
 // Configure Stimulus development experience
-application.debug = false
+application.debug = true
```

Comment:
do you need this in all environemnts? Perhaps you add this config only in development.rb

Reply:
I think I can tell  Stimulus that we are on development by adding a meta tag int the layout:

```ruby
<% if Rails.env == 'development' %>
  <%= tag :meta, name: :rails_env, content: Rails.env %>
<% end %>
```